# Marker pixels in order R0, G0, B0, R1, ...
uint8[] pixels
